### ISSUE TITLE
using UNIX timestamp is not working

### ISSUE THREAD

**Issue by csk647** [tradingview/lightweight-charts#460]
Hi, thanks for your awesome library. : )
I have some problems.
Here is my question. 

**WHAT I WANT**
my raw time format is like this. `2018-03-25T09:00:00.000+0000`
I want to show the candlestick by 1-minute, 3-minute, 1-hour, 1-day.

**PROBLEM**
When I setData of candlestick chart, I tried to change them to `UNIX time` or `2018-03-25`.
`2018-03-25` format is working at safari and chrome but only creating candle by 1-day.`
`UNIX time` is never working at safari. So that, I also can't see that chart in IPhone.(Xr).

**WHAT I DID**
1.
```
timeScale: {
    secondVisible: false,
     timeVisible: true,
     visible: true
}
```

2. change to UNIX timestamp  `new Date(my raw date data).getTime()`
3. change to YYYY-MM-DD
4. change to YYYY-MM-DD HH:MM etc...

how can I show candlestick at safari and chrome both? also create candle by minutes or hour?

**Comment by timocov** [comment]
Can you please create a repro for you issue?

**Comment by csk647** [comment]
you mean, repository?

**Comment by csk647** [comment]
yes, Chart data from back-end is constantly being updated in real time. I want to add or update candle chart every 30 seconds.(with using async await + setTimeout)

whenever the data is set, I'm using a function that data could change format.
like,

```
const remakeDate = ({
  openPrice, 
  highPrice,
  lowPrice,
  closePrice,
  date,
}) => ({
  open: openPrice,
  high : highPrice,
  low: lowPrice,
  close: closePrice,
  time: newDate(date).getTime(),
})
```

this.

And if the chart already be there, just do 
```
setData(chartData.map((el) => remakeData(el)))
```

**Comment by csk647** [comment]
Is this enough...? : )

**Comment by timocov** [comment]
Actually, no. I can't run this sample, I can't see the issue and I can't debug it. Look at `https://github.com/tradingview/lightweight-charts/issues/355#issue-569803110` for sample of good repro. We don't need to guess what the `newDate` function is, we don't need to guess what the data you have and pass to `remakeDate` and so on - just sample of issue, which we can run locally and debug it.
